core/sis-feature/src/test/java/org/apache/sis/feature/builder/AssociationRoleBuilderTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.sis.feature.builder;
+
+import org.apache.sis.util.iso.SimpleInternationalString;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import java.util.Locale;
+import org.apache.sis.feature.DefaultAssociationRole;
+import org.apache.sis.feature.DefaultFeatureType;
+import org.apache.sis.internal.simple.CitationConstant;
+import org.apache.sis.referencing.NamedIdentifier;
+import org.apache.sis.setup.GeometryLibrary;
+import org.apache.sis.util.iso.DefaultNameFactory;
+import org.apache.sis.xml.IdentifierSpace;
+import org.apache.sis.xml.XLink;
+import org.opengis.util.InternationalString;
+
+/**
+ * Unit tests for class {@link AssociationRoleBuilder}.
+ *
+ * @date 2017-09-22
+ * @see AssociationRoleBuilder
+ *
+ **/
```

Comment:
While javadoc comment starts with two stars (`/**`), it usually ends in the usual one-start way (`*/`).

Reply:
Changed it and going to add it in the future with one single asterisk.